ush/machine/wcoss2.yaml

```diff
@@ -27,11 +28,27 @@ platform:
   FIXsfc: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/fix/fix_sfc_climo
   FIXshp: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/NaturalEarth
   EXTRN_MDL_DATA_STORES: hpss aws nomads
+  COMINgfs: /lfs/h1/ops/prod/com/gfs/v16.3
+  COMINgefs: /lfs/h1/ops/prod/com/gefs/v12.3
```

Comment:
Is it possible to handle this using `compath.py` and put it in the data section?

Reply:
I am not sure. I don't have much experience in `compath.py`. Do you have any idea?

Reply:
Currently COMINext serves this purpose and it is set with compath.py on wcoss2. It would need some work if two ics/lbcs sources are used e.g. ensemble members using different sources. So I guess we can ignore this for now.